.github/workflows/build.yaml

```diff
@@ -167,24 +180,30 @@ jobs:
         #   kind-k8s-version: [1.22.17, 1.23.17, 1.24.13, 1.25.9, 1.26.4]
         #   vault-version: [1.11.10, 1.12.6]
         exclude:
+          # vault version 1.11.10
           - kind-k8s-version: 1.26.4
+            vault-version: 1.11.10
           - kind-k8s-version: 1.25.9
+            vault-version: 1.11.10
           - kind-k8s-version: 1.24.13
+            vault-version: 1.11.10
           - kind-k8s-version: 1.23.17
+            vault-version: 1.11.10
           - kind-k8s-version: 1.22.17
-        include:
```

Comment:
As `matrix` has an additional "dimension", I think it's shorter to drop `include` and only do `exclude` combinations of `vault-version` and `kind-k8s-version`. Otherwise, `include` would need to be added `integration-test` to each item, and the list would be long.